packages-exp/auth-exp/src/core/errors.ts

```diff
@@ -123,223 +123,240 @@ export const enum AuthErrorCode {
+      'User can only be linked to one identity for the given provider.',
+    [AuthErrorCode.QUOTA_EXCEEDED]:
+      "The project's quota for this operation has been exceeded.",
+    [AuthErrorCode.REDIRECT_CANCELLED_BY_USER]:
+      'The redirect operation has been cancelled by the user before finalizing.',
+    [AuthErrorCode.REDIRECT_OPERATION_PENDING]:
+      'A redirect sign-in operation is already pending.',
+    [AuthErrorCode.REJECTED_CREDENTIAL]:
+      'The request contains malformed or mismatching credentials.',
+    [AuthErrorCode.SECOND_FACTOR_ALREADY_ENROLLED]:
+      'The second factor is already enrolled on this account.',
+    [AuthErrorCode.SECOND_FACTOR_LIMIT_EXCEEDED]:
+      'The maximum allowed number of second factors on a user has been exceeded.',
+    [AuthErrorCode.TENANT_ID_MISMATCH]:
+      "The provided tenant ID does not match the Auth instance's tenant ID",
+    [AuthErrorCode.TIMEOUT]: 'The operation has timed out.',
+    [AuthErrorCode.TOKEN_EXPIRED]:
+      "The user's credential is no longer valid. The user must sign in again.",
+    [AuthErrorCode.TOO_MANY_ATTEMPTS_TRY_LATER]:
+      'We have blocked all requests from this device due to unusual activity. ' +
+      'Try again later.',
+    [AuthErrorCode.UNAUTHORIZED_DOMAIN]:
+      'The domain of the continue URL is not whitelisted.  Please whitelist ' +
+      'the domain in the Firebase console.',
+    [AuthErrorCode.UNSUPPORTED_FIRST_FACTOR]:
+      'Enrolling a second factor or signing in with a multi-factor account requires sign-in with a supported first factor.',
+    [AuthErrorCode.UNSUPPORTED_PERSISTENCE]:
+      'The current environment does not support the specified persistence type.',
+    [AuthErrorCode.UNSUPPORTED_TENANT_OPERATION]:
+      'This operation is not supported in a multi-tenant context.',
+    [AuthErrorCode.UNVERIFIED_EMAIL]:
+      'The operation requires a verified email.',
+    [AuthErrorCode.USER_CANCELLED]:
+      'The user did not grant your application the permissions it requested.',
+    [AuthErrorCode.USER_DELETED]:
+      'There is no user record corresponding to this identifier. The user may ' +
+      'have been deleted.',
+    [AuthErrorCode.USER_DISABLED]:
+      'The user account has been disabled by an administrator.',
+    [AuthErrorCode.USER_MISMATCH]:
+      'The supplied credentials do not correspond to the previously signed in user.',
+    [AuthErrorCode.USER_SIGNED_OUT]: '',
+    [AuthErrorCode.WEAK_PASSWORD]:
+      'The password must be 6 characters long or more.',
+    [AuthErrorCode.WEB_STORAGE_UNSUPPORTED]:
+      'This browser is not supported or 3rd party cookies and data may be disabled.'
+  };
+}
+export interface AuthErrorMap {}
+
+export interface ErrorMapRetriever extends AuthErrorMap {
+  (): ErrorMap<AuthErrorCode>;
+}
+
+function _prodErrorMap(): ErrorMap<AuthErrorCode> {
+  return {} as ErrorMap<AuthErrorCode>;
+}
+
+export const verboseErrorMap: AuthErrorMap = _verboseErrorMap;
```

Comment:
nit: debugErrorMap? debug vs prod makes more senes to me than verbose vs prod, alternatively you can do long vs short, or verbose vs concise